uw-research-computing/python-jobs.md

```diff
@@ -70,24 +82,49 @@ packages to a folder and zip those files to return to the submit server.
 > install of Python; if you want to add packages to a pre-existing
 > package folder, there will be notes below in boxes like this one.
 
+Preliminary Step: Choose a Linux Version to Build On
+----------------------------------
+
+As of August 2022, the newest hardware in the HTC system is running a newer version of Linux, 
+CentOS Stream 8. Our older hardware is still running on CentOS 7 and will gradually be 
+transitioning over to the new operating system. You can see the number of available cores under each operating 
+system here: [HTC Operating System Transition](/uw-research-computing/os-transition-htc.html)
+
+There are two approaches to running on our pool: 
+
+- **compile on CentOS 7, run on both versions of CentOS**: This is the option you should 
+choose if you want to have as much capacity as possible available to you and you 
+are willing to do a little more testing of your jobs. 
```

Comment:
This seems to be true unless the job does anything that touches the SSL libraries. Most common reason would be if the job does anything with HTTPS URLs. The error will be something like
```
urllib3.exceptions.SSLError: Can't connect to HTTPS URL because the SSL module is not available.
```